client/.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package*.json
```

Comment:
I meant the `.gitignore` in the root directory not the `client`. But remove it and don't add it anywhere.